static_src/components/user_provider.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Information on the currently logged in user is something that I think can safely be made global to the entire application. Otherwise, for every component that cares about the current user, we'd have to add a change listener to the store, set state, etc. That gets a bit cumbersome.

In general I think we should be moving towards having less direct store access in components and more props being passed down from container components. I think that was the direction marco wanted to start moving in as well ( see the **container component** section in CONTRIBUTING.md).

In this case, I wasn't sure exactly what component would be the container (likely `Users`), and the currentUser prop would probably end up getting passed down through about 5 components. We can definitely revisit this later whenever we refactor the code to make use of containers more heavily!